src/tools/rbd/action/MirrorPool.cc

```diff
@@ -399,6 +503,10 @@ Shell::Action action_info(
   {"mirror", "pool", "info"}, {},
   "Show information about the pool mirroring configuration.", {},
   &get_info_arguments, &execute_info);
+Shell::Action action_status(
+  {"mirror", "pool", "status"}, {},
```

Comment:
Perhaps the high-level summary could use separate class methods and avoid image name lookups to be a lot faster when there are tons of images. With batching maybe this isn't worth optimizing right now though.